src/Portal/Portal.d.ts

```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 
 export interface PortalProps {
   children: React.ReactElement<any>;
-  container?: object | Function;
-  onRendered?: Function;
+  container?: React.ReactInstance | (() => React.ReactInstance);
+  onRendered?: () => any;
```

Comment:
`() => void`

Reply:
Oh, sorry. I just saw your comment:

> `() => void` actually makes sense. However, `onRendered` is passed to React lifecycle `forceUpdate` and at the moment `forceUpdate` signature is:
> 
> ```ts
> forceUpdate(callBack?: () => any): void;
> ```
>
> So I think `onRendered` should match `callBack` type here.

`() => void` is definitely the correct type, and it is assignable to `() => any`, so it doesn't introduce any incompatibilities to do the right thing. I will open a PR for the React types to fix their `forceUpdate` type 🙂

Reply:
Ok, then I updated it to `() => void`